installers/agent/osx/Info.plist

```diff
@@ -53,6 +53,8 @@
         <dict>
             <key>ClassPath</key>
             <string>$APP_PACKAGE/agent-bootstrapper.jar</string>
+            <key>JVMVersion</key>
```

Comment:
This doesn't matter. It's not used. We don't use the Java key for this dict. We use [JavaX](https://github.com/gocd/gocd/blob/master/build/osx/JavaApplicationStub64#L225), a made up key, almost. Using the Java key for the dict ties it to Apple's Java and that causes more problems than it solves (especially since Apple stopped supplying Java).